quartz/plugins/transformers/ofm.ts

```diff
@@ -260,6 +260,11 @@ export const ObsidianFlavoredMarkdown: QuartzTransformerPlugin<Partial<Options>
                     value: alias ?? fp,
                   },
                 ],
+                data: {
+                  hProperties: {
+                    class: alias ? "alias" : "",
+                  },
+                },
```

Comment:
this doesn't cover cases where it's a regular markdown link, is that fine for your use case?

Reply:
Good point. This should be resolved in the latest commit